src/plugin.js

```diff
@@ -44,25 +48,24 @@ const errors = {
 /**
  * Initializes the plugin
  */
-const vr = function(options) {
-  const player = this;
+const initPlugin = function(player, options) {
 
   // don't initialize twice
-  if (this.usingPlugin('vr')) {
+  if (player.vr && player.vr.currentProjection) {
     videojs.log.warn('videojs-vr is already intialized, not going to initialize again');
     return;
   }
 
   const settings = videojs.mergeOptions(defaults, options || {});
-  const videoEl = this.el().getElementsByTagName('video')[0];
-  const container = this.el();
-  let currentProjection = settings.projection;
-  let movieMaterial;
-  let movieGeometry;
-  let movieScreen;
-  let controls3d;
-  let scene;
-  let cameraVector;
+  const videoEl = player.el().getElementsByTagName('video')[0];
+  const container = player.el();
+  const defaultProjection = settings.projection;
+  const oldBigPlayButton = player.BigPlayButton;
```

Comment:
Also, we probably don't need to store the old one. If it wasn't removed, we can just get the reference via `getChild` and then set it `reset`. One less thing to track ourselves here.